src/fc/qemu/agent.py

```diff
@@ -50,6 +50,12 @@ def _handle_consul_event(event):
 
 EXECUTABLE = sys.argv[0]  # make mockable
 
+MAINTENANCE_TEMPFAIL = 75  # Retry shortly
+MAINTENANCE_POSTPONE = 69  # Retry when the directory schedules a new window
```

Comment:
We do not postpone anywhere in this code so far. I'm okay with leaving this here for reference in case it is needed laterm but want to ensure that this is intended.